### PR TITLE
feat: use cases section

### LINKED ISSUE #107: [FEATURE] Build the Use Cases section
### Detailed description

image

### Context

you can see the design here: https://www.figma.com/file/rmCKKIsjSWCH0q6lObDQC8/Dev-Protocol-design?node-id=0%3A1

### Possible implementation

Use the same font Whyte Inktrap

### Additional information

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by krishguptadev**
fixes #107 

<a href="https://gitpod.io/#https://github.com/web3community/devprotocol.xyz/pull/109"></a>

### CODE REVIEW

**Review by vinzvinci — DISMISSED** [review]

File: src/components/Creators/UseCases.astro [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        <p class="max-w-para">Offer exclusive perks for patrons to earn revenue more actively. Power up your own creator economy.</p>
+      </div>
+      <div class="flex flex-col w-full justify-space-evenly p-4 m-4 ml-0 pl-0">
+        <img width="100" height="100" src="/assets/creator.png" alt="">
+        <h4 class="font-display font-black text-4xl mt-5 mb-10">Governance</h4>
+        <p class="max-w-para">Offer exclusive perks for patrons to earn revenue more actively. Power up your own creator economy.</p>
+      </div>
+    </div>
+  </div>
+</section>
```
Comment (krishguptadev) [inline src/components/Creators/UseCases.astro]:
```suggestion
</section>

```

File: src/components/Creators/UseCases.astro [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      </div>
+      <div class="flex flex-col w-full justify-space-evenly p-4 m-4 ml-0 pl-0">
+        <img width="100" height="100" src="/assets/creator.png" alt="">
+        <h4 class="font-display font-black text-4xl mt-5 mb-10">Perks</h4>
+        <p class="max-w-para">Offer exclusive perks for patrons to earn revenue more actively. Power up your own creator economy.</p>
+      </div>
+      <div class="flex flex-col w-full justify-space-evenly p-4 m-4 ml-0 pl-0">
+        <img width="100" height="100" src="/assets/creator.png" alt="">
+        <h4 class="font-display font-black text-4xl mt-5 mb-10">Governance</h4>
+        <p class="max-w-para">Offer exclusive perks for patrons to earn revenue more actively. Power up your own creator economy.</p>
```
Comment (krishguptadev) [inline src/components/Creators/UseCases.astro]:
```suggestion
        <p class="max-w-para">Share ownership of your assets with your team/stakeholders for efficient and secure governance..</p>
```

File: src/components/Creators/Hero.astro [hunk 3]
```diff
@@ -5,8 +5,7 @@
     justify-center
```
Comment (krishguptadev) [inline src/components/Creators/Hero.astro:5]:
```suggestion
    justify-space-between
```

File: src/components/Creators/Hero.astro [hunk 4]
```diff
@@ -5,8 +5,7 @@
...
     text-center text-lg
     lg:text-left
-    py-16
-    lg:py-36
+    py-10
     px-4
```
Comment (krishguptadev) [inline src/components/Creators/Hero.astro:9]:
```suggestion
    px-10
```

File: src/components/Creators/Hero.astro [hunk 5]
```diff
@@ -2,14 +2,13 @@
   class="
     flex flex-col
     items-center
-    justify-center
+    justify-between
```
Comment (krishguptadev) [inline src/components/Creators/Hero.astro:5]:
```suggestion
    justify-evenly
```

**Review by vinzvinci — APPROVED** [review]

**Review by shuklaritvik06 — APPROVED** [review]

File: src/components/Creators/UseCases.astro [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+      </div>
+      <div class="flex flex-col w-full justify-evenly p-4 m-4 ml-0 pl-0">
+        <img width="100" height="100" src="/assets/creator.png" alt="">
+        <h4 class="font-display font-black text-4xl mt-5 mb-10">Governance</h4>
+        <p class="max-w-para">Share ownership of your assets with your team/stakeholders for efficient and secure
+          governance.</p>
+      </div>
+    </div>
+  </div>
+</section>
```
Comment (Panquesito7) [inline src/components/Creators/UseCases.astro:25]: This is missing an endline. I think it was already fixed, though. 🤔

**Pull request merged**